app/assets/stylesheets/responsive/_header_layout.scss

```diff
@@ -120,6 +120,14 @@
 
 }
 
+.no-js #banner {
+  @include respond-min( $main_menu-mobile_menu_cutoff ){
+    #logged_in_bar {
+      margin-left: 3em;
```

Comment:
This isn't a great fix because some sites may have much shorter navs:

![screen shot 2017-09-01 at 13 04 24](https://user-images.githubusercontent.com/282788/29969031-45f4b0b8-8f16-11e7-9d61-f5709741ca32.png)

I think we can fix this for all eventualities with `margin-left: 70%;`:

![screen shot 2017-09-01 at 13 04 18](https://user-images.githubusercontent.com/282788/29969052-5bf6a574-8f16-11e7-88fd-b534662f1703.png)

Reply:
`margin-left: 40%` seems safer as 70% with a non-pro user logged in does this:

<img width="1012" alt="screen shot 2017-09-01 at 13 21 03" src="https://user-images.githubusercontent.com/27760/29969650-ee9b9f40-8f18-11e7-8e03-7e5cd440e719.png">

the first value that works is 20%, the last is 60% so choosing the one in the middle!

Reply:
more scientifically, `50%` seems right, if the nav bar is shorter than 12 characters, that's different problem:

<img width="1080" alt="screen shot 2017-09-01 at 13 41 42" src="https://user-images.githubusercontent.com/27760/29970232-ab070988-8f1b-11e7-84e4-64bf6b099aaa.png">